### PR TITLE
Add Use cases to Developer Guide

### PR DESCRIPTION
**Pull request by ThePrevailingOne**

### CODE REVIEW

**Review by Graphcalibur — CHANGES_REQUESTED** [review]
I think that more use cases should be added.
Also, should we add more terms to the glossary?

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -285,14 +285,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
```
[the remark this answers is not in the document]
Comment (ThePrevailingOne) [reply docs/DeveloperGuide.md]: Already changed 👍

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -283,16 +283,71 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+   Use case ends.
+
+**Use case: Add a new company**
+
+**MSS**
+
+1.  User requests to add a new company with details given
+2.  InternBuddy creates a new company
+3.  InternBuddy assign a tag to the company with the company name as the tag
```
Comment (Graphcalibur) [inline docs/DeveloperGuide.md:332]: > assign a tag to the company with the company name as the tag

What does this mean?

**Review by Graphcalibur — CHANGES_REQUESTED** [review]

**Comment by ThePrevailingOne** [comment]
> > assign a tag to the company with the company name as the tag
>
> What does this mean? 

since currently the company doesn't have any relation to the tag, the tag used by event and contact person is not associated to the company if I don't assign it.

**Review by Graphcalibur — APPROVED** [review]
LGTM then!

**Pull request merged**